tests/details.rs

```diff
@@ -15,15 +12,20 @@ fn test_string_escape() {
     ];
 
     let starlark = serde_starlark::to_string(strings).unwrap();
-    assert_snapshot!(starlark, @r###"
-    [
-        "\a \b \f \n \r \t \v \\",
-        "Have you read \"To Kill a Mockingbird?\"",
-        "Yes, it's a classic.",
-        "AД界😀",
-        "\0\x000 \1\x010 \16\x0E0 \177\1770 \u0080",
-    ]
-    "###);
+    k9::snapshot!(
+        starlark,
+        r#"
+[
+    "\\a \\b \\f \
+ \\r \\t \\v \\\\",
```

Comment:
@aaronabramov the treatment of backslashes here seems very incorrect and unexpected to me. The content of `starlark` is `"\a \b \f \n \r \t \v \\"` in Display representation / `"\"\\a \\b \\f \\n \\r \\t \\v \\\\\""` in Debug representation. So some things stand out:

- I would expect to see _either_ `\"\\a` _or_ `"\a` in the snapshot, depending on whether the debug or display is the one being rendered. I would definitely not expect to see `"\\a` as seen here.

- There is no newline character in this part of `starlark`, and yet k9 is adding a newline into the snapshot, after one of the backslash characters.

- Only some of the backslashes in the input appear escaped in the snapshot.